### PR TITLE
fix(build): SILE version might differ from the system default

### PR DESCRIPTION
**Pull request by Omikhleia**
Maybe there's a way to be more clever?

### CODE REVIEW

**Comment by Omikhleia** [comment]
> Maybe there's a way to be more clever?

I dunno but rebuilt lots of my resilient docker images on different machines recently, and this did the trick.

File: build/Dockerfile [hunk 1]
```diff
@@ -17,18 +17,19 @@ RUN pacman --noconfirm -S ghostscript
...
-# COPY ./build/fonts/OCRB.otf /.fonts
 
-# Should be done
-RUN luarocks list
+# Just to check the version on first run
 RUN sile --version
 
+# Resilient packages and classes for SILE
+# Try to deduce the Lua version used by SILE, which might be different from the system Lua
+RUN v=$(sile --quiet -e 'v = (_VERSION):gsub("Lua ", ""); print(v); os.exit()'); echo $v > /LVER
```
Comment (alerque) [inline build/Dockerfile:30]:
```
$sile -q -e 'print(SILE.lua_version); os.exit()'
5.1
```

Comment (Omikhleia) [reply build/Dockerfile:30]: Wow :heart:

**Comment by Omikhleia** [comment]
Neat!

```
# New 0.15.5-based archlinux image with SILE (`SILE v0.15.5 (LuaJIT 2.1.1723675123) [Rust]`)
# with everything freshly rebuilt
$ resilient-new -q -e 'print(SILE.lua_version); os.exit()'
5.1
# 0ld production workflow on custom (modified) 0.14 build (`SILE v0.14.11.r21-g3b0ea5a (Lua 5.4)`))
# on a fairly old archlinux image
$ resilient -q -e 'print(SILE.lua_version); os.exit()'
5.4
```

File: build/Dockerfile (hunk 1 above)
Comment (alerque) [reply build/Dockerfile:30]: The Dockerfile shell stuff can be simplified too ;-)

```dockerfile
RUN sile -qe 'print(SILE.lua_version)' > /LVER
```

**Comment by alerque** [comment]
The `SILE.lua_version` thing has been around since v0.10.10, but the quiet flag didn't get added until v0.14.9 which makes it much easier to reach for. In v0.15+ the flags can be compressed `-qe`, but that doesn't help you if you're potentially querying v0.14 with the same commands.

I'm considering adding some flag or subcommand (`sile lua ...`?)  that would assume no input document and just run Lua stuff without needing the `os.exit()` at the end to terminate before trying to go on to a document. Not just for this little Lua version trick of course, but I find myself querying a number of things from inside SILE other than while rendering documents.

**Pull request merged**